Enformer.pm

```diff
@@ -0,0 +1,156 @@
+=head1 LICENSE
+Copyright [1999-2015] Wellcome Trust Sanger Institute and the EMBL-European Bioinformatics Institute
+Copyright [2016-2023] EMBL-European Bioinformatics Institute
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+     http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+=head1 CONTACT
+ Ensembl <http://www.ensembl.org/info/about/contact/index.html>
+=cut
+
+=head1 NAME
+ Enformer
+
+=head1 SYNOPSIS
+ 
+ mv Enformer.pm ~/.vep/Plugins
+ ./vep -i variations.vcf --plugin Enformer,file=Enformer_grch38.vcf.gz
+
+
+=head1 DESCRIPTION
+ This is a plugin for the Ensembl Variant Effect Predictor (VEP) that adds pre-calculated scores from Enformer.
+
+ This Plugin is available for GRCh37 and GRCh38
+ 
+ Please cite the Enformer publication alongside the VEP if you use this resource:
+ https://www.nature.com/articles/s41592-021-01252-x
+
+ Enformer scores can be downloaded from https://ftp.ensembl.org/pub/current_variation/Enformer for GRCh37 and GRCh38.
+
+ The plugin can then be run as default:
+ ./vep -i variations.vcf --assembly GRCh38 --plugin Enformer,file=/path/to/Enformer/data.vcf.gz
+
+ or run with option to only retrieve the SAD score 
```

Comment:
Maybe it would be useful to explicitly state what SAD means:
```suggestion
 or run with option to only retrieve the SAD (SNP Activity Difference) score 
```